### PR TITLE
[prometheus-postgres-exporter] rollout deployment from secret update

### PR DESCRIPTION
**Pull request by rasta-rocket**
#### What this PR does / why we need it

Hey folks :wave:

I made that little PR to allow the deployment to rollout restart (through the labels update) when the password in the secret is updated.

Don't hesitate to ping if you any question 😉 

Cheers ☀️ 

#### Which issue this PR fixes

- None

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-postgres-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "v0.15.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 6.2.0
+version: 6.2.1
```
Comment (zeritti) [inline charts/prometheus-postgres-exporter/Chart.yaml:5]: As an enhancement/new feature, I'd suggest to make it a minor change.
```suggestion
version: 6.3.0
```

**Review by zeritti — CHANGES_REQUESTED** [review]
Thank you, @rasta-rocket, for your PR. Please, see my comments below.

File: charts/prometheus-postgres-exporter/Chart.yaml (hunk 1 above)
Comment (rasta-rocket) [reply charts/prometheus-postgres-exporter/Chart.yaml]: @zeritti Thank you for the review the change has been pushed

**Review by zeritti — APPROVED** [review]
Thank you, @rasta-rocket, LGTM.

**Pull request merged**